versioned_docs/version-2.x/appendices/migration-guides/index.mdx

```diff
@@ -14,6 +14,15 @@ import TabItem from "@theme/TabItem";
 
 ## `2.24.0` -> `2.25.0`
 
+### Drop support of Node 14
+
+In this version we've **dropped support for Node 14**
+(<abbr title="End Of Life">EOL</abbr>: 2023-04-30). The minimum required version
+is now Node 16.
+
+You need to check your project dependencies for compatibility with at least
+Node 16.
```

Comment:
It's not super clear as to what actions integrators should take to achieve this.

Reply:
Added instruction to check dependencies versions on npm website, but don't know what to add

Reply:
https://deploy-preview-695--heuristic-almeida-1a1f35.netlify.app/docs/2.x/appendices/migration-guides/#drop-support-of-node-14